### PR TITLE
Delete and Edit tag test

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 719ce8c6fdb98612d1d40c2cb23b1ff2e0469d00:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/cloud/test_stack.py [hunk 1]
```diff
@@ -58,3 +58,11 @@ def test_outputs_link(stack):
 
 def test_resources_link(stack):
     stack.nav_to_resources_link()
+
+
+def test_edit_tags(stack):
+    stack.edit_tags("Cost Center *", "Cost Center 001")
```
Comment (dajohnso) [inline cfme/tests/cloud/test_stack.py]: We should add a assert to verify this is displayed on the stack summary screen

**Comment by dajoRH** [comment]
Lint report for commit 0da00c34a122666ed1467aa6881b0b818df6d915:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 57378c832348791d950d18fa8b12838c1be72f2f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5b9ed124293d5b5afdd58969ee6b62a46e5380ca:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**